server/routes/incidentDetails/incidentDetailsPage.ts

```diff
@@ -707,6 +707,7 @@ const setUpRedirectForEditError = (res: Response, prisonerNumber: string, error:
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 const setUpRedirectForCreationError = (res: Response, prisonerNumber: string, error: any) => {
   logger.error(`Failed to post incident details for draft adjudication: ${error}`)
+  // TODO: @steve not sure this URL is correct - shouldn't it be /id ?
   res.locals.redirectUrl = `/incident-statement/${prisonerNumber}`
```

Comment:
Yep - I copied this stuff from the original so i think it was always wrong.

Reply:
I'll change it in the next PR - obviously wasn't used very much if it was always wrong and nobody noticed!